action.yml

```diff
@@ -38,6 +38,14 @@ inputs:
     description: 'Override values for the Helm chart in YAML string'
     required: false
     default: ''
+  values-inline:
+    description: 'Override values for the Helm chart in inline form'
+    required: false
+    default: ''
+  delete-sts:
+    description: 'Configure either Weaviate Statefulset should be removed'
+    required: false
+    default: 'true'
```

Comment:
Just thinking...shouldn't this be false? I mean, if we consider in how many upgrade jumps we are going to delete the STS (vs) in how many we don't need to, it's a 1 vs Many. So, it might make more sense to default this to false and when we upgrade from non-raft to raft we pass the value to true. Will that behavior impact your upgrade journey chaos-pipeline to much?

Reply:
Yeah, I was thinking about this also that it should `false` but in our original script it's `true` by default, that's why I left it, so that there will be no breaking changes, but we can change this if you agree?

Reply:
> Will that behavior impact your upgrade journey chaos-pipeline to much?
Not that much, but I'm more thinking about our users

Reply:
Yes, let's change it to false by default and pass true if required to upgrade from 1.24 to 1.25

Reply:
Done